runbot_gitlab/runbot_repo.py

```diff
@@ -146,7 +147,16 @@ def set_gitlab_ci_conf(token, gitlab_url, runbot_domain, repo_id):
 
 class RunbotRepo(models.Model):
     _inherit = "runbot.repo"
-    uses_gitlab = fields.Boolean('Use Gitlab')
+
+    uses_gitlab = fields.Boolean(string='Use Gitlab')
+
+    mr_only = fields.Boolean(
+        string="MR Only",
+        help="Build only merge requests and skip regular branches")
```

Comment:
By not setting the default to True, you are changing the default behaviour of this module. Existing uses might be confused. Would you consider setting the defaults so as to preserve the previous behaviour?